nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteProcess.java

```diff
@@ -187,7 +187,7 @@ public void shutdownExecutor() {
             executor.shutdown();
         } finally {
             if (this.externalProcess.isAlive()) {
-                this.getLogger().warn("Interrupting external process since it did not finished at the time stop Processor was initiated.");
+                this.getLogger().warn("Process hasn't terminated, forcing the interrupt");
```

Comment:
Do we want to file a WARN bulleting? It shows up in the UI. E.g. is there a good way at all to kill a process which doesn't have any external interrupt handles exposed? Lower to INFO maybe?